core/templates/dev/head/i18n.js

```diff
@@ -37,6 +37,27 @@ oppia.constant('DEFAULT_TRANSLATIONS', {
   I18N_SIGNUP_LOADING: 'Loading'
 });
 
+oppia.factory('customLoader', [
```

Comment:
This is service used by i18n. The function inside is called internally after the language that should be used is known.